pkg/flow/grpc-namespaces.go

```diff
@@ -318,7 +318,7 @@ func (flow *flow) DeleteNamespace(ctx context.Context, req *grpc.DeleteNamespace
 		return nil, err
 	}
 
-	flow.deleteNamespaceSecrets(ns)
+	// TODO: need to delete secretes related to namespace here.
```

Comment:
Is this necessary? I think foreign key cascade should take care of it. 

If not, be sure to finish this TODO. I don't consider this mergeable if we're leaking records.